src/metalperformanceshaders.cs

```diff
@@ -1775,6 +1784,84 @@ interface MPSTemporaryImage {
 		nuint ReadCount { get; set; }
 	}
 
+	[Introduced (PlatformName.MacCatalyst, 13, 0)]
+	[TV (13,0), Mac (10,15), iOS (13,0)]
+	[BaseType (typeof(NSObject))]
+	interface MPSPredicate
+	{
+		[Export ("predicateBuffer", ArgumentSemantic.Retain)]
+		IMTLBuffer PredicateBuffer { get; }
+
+		[Export ("predicateOffset")]
+		nuint PredicateOffset { get; }
+
+		[Static]
+		[Export ("predicateWithBuffer:offset:")]
+		MPSPredicate Create (IMTLBuffer buffer, nuint offset);
+
+		[Export ("initWithBuffer:offset:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (IMTLBuffer buffer, nuint offset);
+
+		[Export ("initWithDevice:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (IMTLDevice device);
+	}
+
+	interface IMPSHeapProvider {}
+
+	[Protocol]
+	[BaseType (typeof(NSObject))]
+	[Model]
```

Comment:
```suggestion
	[Model (AutoGeneratedName = true)]
```

For any new `Model` please add the `AutoGeneratedName = true`

Reply:
Got it!